server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/DefaultMirroringServicePlugin.java

```diff
@@ -97,8 +127,9 @@ public MirrorSchedulingService mirroringService() {
     @Override
     public String toString() {
         return MoreObjects.toStringHelper(this)
-                          .add("configType", configType())
-                          .add("target", target())
+                          .omitNullValues()
+                          .add("configType", configType().getName())
+                          .add("target", pluginTarget)
```

Comment:
It was intentional and I thought it was safe because `target(CentralDogmaConfig)` is called right after the plugin is initialized. `pluginTarget` may be not null for most of the object's lifecycle.